style.css

```diff
@@ -104,3 +104,380 @@ button {
+  color: #344563;
+}
+
+.tags {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  padding: 4px 12px;
+  gap: 8px;
+  width: 203px;
+  height: 24px;
+}
+
+.html_but {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.html_text {
+  width: 29px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.css_but {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.css_text {
+  width: 21px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.java_but {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: 89px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.java_text {
+  width: 65px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.but_container {
+  padding-bottom: 100px;
+  width: 295px;
+  height: 72px;
+}
+
+.Action_but {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 12px;
+  gap: 10px;
+  width: 130px;
+  height: 48px;
+  left: 0;
+  border: 2px solid #6070ff;
+  border-radius: 8px;
+  margin-top: 5px;
+}
+
+.text_project {
+  width: 100px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.About_section {
+  position: absolute;
+  width: 375px;
+  height: 1263px;
+  left: 40px;
+  top: 4500px;
+}
+
+.About_text {
+  width: 327px;
+  height: 101px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.software_text {
+  width: 327px;
+  height: 144px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 12px;
+}
+
+.resume_but {
+  padding-bottom: 100px;
+  width: 295px;
+  height: 72px;
+}
+
+.Action_but2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 12px;
+  gap: 10px;
+  width: 170px;
+  height: 55px;
+  left: 0;
+  border: 2px solid #6070ff;
+  border-radius: 8px;
+  margin-top: -10px;
+}
+
+button:hover {
+  background: orangered;
+}
```

Comment:
- [ ] Good job for implemented the the button interactions (enable, hover, etc.), but kindly i recommend you implement  button interaction just like the Figma design.

 
![Capture](https://user-images.githubusercontent.com/93598189/220694582-0140cf03-0c55-4d0b-84cb-d2647ef8cca4.PNG)